modules/ROOT/partials/misc/admon-requires-6.3v.adoc

```diff
@@ -0,0 +1 @@
+NOTE: This feature is only available for {productname} 6.3 and later.
```

Comment:
```suggestion
NOTE: This feature is only available for {productname} 6.3 and later.

```
Adding a blank line.

Consequent to @ephox-mogran’s [comment](https://github.com/tinymce/tinymce-docs/pull/2519#issuecomment-1254372895), I’m taking the leap from [commentary](https://github.com/tinymce/tinymce-docs/pull/2519#issuecomment-1253564668) to concrete suggestion.